requirements.txt

```diff
@@ -1,8 +1,6 @@
 pbr!=2.1.0,>=2.0.0 # Apache-2.0
 uvloop==0.11.3
 ujson==1.35
-requests==2.20.1
-dill==0.2.8.2
 iso8601==0.1.12
-aiohttp==3.4.4
-pytest-aiohttp==0.3.0
+h11==0.8.1
+pytest-asyncio==0.9.0
```

Comment:
can we get rid of most of these? we absolutely do not need a high performance socket. ujson appears to only be used for tests. don't know about the rest. if they aren't affecting function load times then maybe we can play ball or delay the import where they're used, however just having the requirement does add time to the function build loop to install these and the fewer the better with imports...

Reply:
The rest of dependencies are necessary, they wouldn’t go anyway. The do not affect the startup time. Pytest it requires by testing framework FDK provides. So, requirements file is in its final form.

Reply:
QQ: Is uvloop the only cpython dep here and is the speedup it gives worth it? Not an expert but comparing experiences with ruby and node I see benefit to only having pure python deps (especially w.r.t security maintainability, portability)

Reply:
uvloop is opt-in dependency for now, if people need them - they'll install it by themselves.